### PR TITLE
Refactoring cloud-card section

### LINKED ISSUE #13: Refactoring cloud-card section
image
- you reassign a lot of styles from ".card" so I think you don't need to use this class.

- fix name "--creditcard-width"

image
- strange width property

- use bg image instead of tag img. Selector - "creditcard-card__background-image"
 
image
- redundant width

image
- do you really need flex here?

image
- fixed height is bad :)

- using flex for "card-balance__item" is redundant. Just replace with text-align: right

image
- checkbox breaks down when width changed. Check component for flexibility of different sizes

### PR DESCRIPTION
**Pull request by nato000**
image

⬇

image

### CODE REVIEW

File: css/components/card/card.css [hunk 1]
```diff
@@ -27,13 +27,13 @@
...
   overflow: hidden;
-  width: calc(var(--creditcard-width) + 10px);
+  width: var(--width);
   padding: 0 20px 0 0;
 }
 
 .card-prewiev {
   display: flex;
-  width: calc(var(--creditcard-width) * var(--quantity) + 25px);
+  width: calc(var(--width) * var(--quantity) + 25px);
```
Comment (frontend-sensei) [inline css/components/card/card.css]: Why do you set width here?

File: css/components/card/card.css [hunk 2]
```diff
@@ -64,13 +64,13 @@
 
 .card-prewiev__btn-left {
   left: 10px;
-  top: calc(0px + var(--creditcard-height) * 0.5);
+  top: calc(0px + var(--height) * 0.5);
```
Comment (frontend-sensei) [inline css/components/card/card.css]: For what "0px"?

File: css/components/card/card.css [hunk 3]
```diff
@@ -64,13 +64,13 @@
...
   transform: translate(-100%, -50%);
 }
 
 .card-prewiev__btn-right {
   right: 0px;
-  top: calc(0px + var(--creditcard-height) * 0.5);
+  top: calc(0px + var(--height) * 0.5);
```
Comment (frontend-sensei) [inline css/components/card/card.css]: Duplicated props. (Line 67)

File: css/components/card/card.css (hunk 2 above)
Comment (nato000) [reply css/components/card/card.css]: idk mb bug

Comment (nato000) [reply css/components/card/card.css]: that's not mine))

**Comment by nato000** [comment]
image

⬇

image

File: css/components/card/card.css [hunk 4]
```diff
@@ -24,16 +23,13 @@
 
 .card-prewiev-wrapper {
   --quantity: 2;
```
Comment (frontend-sensei) [inline css/components/card/card.css]: Redundant var

**Review by frontend-sensei — CHANGES_REQUESTED** [review]

**Pull request merged**